src/applications/personalization/profile/components/ProfileInfoTable.jsx

```diff
@@ -82,7 +84,11 @@ const ProfileInfoTable = ({
   // can be passed directly to a `className` attribute
   const classes = {
     table: ['profile-info-table', className].join(' '),
-    title: [...titleClasses, ...titleClassesMedium].join(' '),
+    title: [
+      'vads-u-font-size--h3',
```

Comment:
Can you move this new class to the `titleClasses` array instead?

Reply:
yep sorry about that